central/declarativeconfig/manager_impl.go

```diff
@@ -310,7 +310,7 @@ func (m *managerImpl) doDeletion(transformedMessagesByHandler map[string]protoMe
 	}
 
 	if err := m.removeStaleHealthStatuses(allProtoIDsToSkip); err != nil {
-		log.Errorf("Failed to delete stale health status entries for declarative config: %v", err)
+		log.Debugf("Failed to delete stale health status entries for declarative config: %v", err)
```

Comment:
I would argue that it's OK to switch to `Error` log level for errors which modify integration health - however, for errors that are only displayed in logs switching to `Debug` level would make them essentially invisible 🤔